src/main/java/duke/parsers/DateTimeParser.java

```diff
@@ -0,0 +1,52 @@
+package duke.parsers;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Class to parse a DateTime object into a String of different format or vice versa
+ */
+public class DateTimeParser {
+
+    /**
+     * Converts DateTime object to String
+     * @param ldt DateTime object to be converted
+     * @return a String representation of the DateTime object in a different format.
+     */
+    public static String dtToString(LocalDate ldt) {
+        DateTimeFormatter outputFormatter = DateTimeFormatter.ofPattern("MMM d yyyy");
+        String formattedDateTime = ldt.format(outputFormatter);
+        return formattedDateTime;
+    }
+
+    /**
+     * COnverts String to DateTime object
+     * @param s String to be converted
+     * @return DateTime object to be returned to store into a Task.
+     */
+    public static LocalDate stringToDT(String s) {
```

Comment:
can consider using "DateTime" here for this method instead or abbreviation to make your naming more consistent since you use DateTime (no acronym/abbreviation) in other places also!